packages/dashboard/src/CdcDashboardComponent.tsx

```diff
@@ -358,7 +357,8 @@ export default function CdcDashboard({ initialState, isEditor = false, isDebug =
         if (Array.isArray(value)) throw Error(`Cannot set active values on urlfilters. expected: ${JSON.stringify(value)} to be a single value.`)
         newSharedFilters[index].queuedActive = value
         // setData to empty object because we no longer have a data state.
-        dispatch({ type: 'SET_DATA', payload: {} })
+        // Freeze object to ensure no side effects.
+        dispatch({ type: 'SET_DATA', payload: Object.freeze({}) })
         dispatch({ type: 'SET_FILTERED_DATA', payload: {} })
```

Comment:
Does this need to use object.freeze as well?

Reply:
I couldn't figure out where a side effect was happening and once I froze the object on line 361 I immediately got an error on line 569. Object.freeze is not needed now for the purposes of this pr but I thought it would be good to leave it to prevent any future side effects on Data. I felt like maybe the FilteredData was less permanent? so, IDK if that same reasoning applies to 362. What do you think?